config.py

```
@@ -0,0 +1,5 @@
+# keys and env variables in code
+HARDCODED_API_KEY = "sk-live-please-rotate-me-1234567890"
+DEFAULT_PASSWORD = "password123"
```

Ruff (0.12.2)

Possible hardcoded password assigned to: "DEFAULT_PASSWORD"

(S105)